### PR TITLE
[CDAP-8269][CDAP-8386] Fix for search/sort/pagination bugs

### PR DESCRIPTION
**Pull request by tonybach**
JIRAs: https://issues.cask.co/browse/CDAP-8269, https://issues.cask.co/browse/CDAP-8386
Build: http://builds.cask.co/browse/CDAP-DRC5629

### CODE REVIEW

File: cdap-ui/app/cdap/components/EntityListView/EntityListHeader/index.js [hunk 1]
```diff
@@ -100,31 +95,28 @@ export default class EntityListHeader extends Component {
...
-                  {option.displayName}
-                  {
-                    this.state.activeSort.fullSort === option.fullSort ?
-                      <span className="fa fa-check float-xs-right"></span>
-                    :
-                      null
-                  }
-                </DropdownItem>
-              );
+              if (index !== 0) {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/EntityListView/EntityListHeader/index.js]: If this is true for all the cases then we might as well have it as,

```
this.state.sortOptions.slice(1).map((option) => {...})
```

Simpler?

**Review by ajainarayanan — COMMENTED** [review]
Minor comment. Otherwise 👍

File: cdap-ui/app/cdap/components/EntityListView/EntityListHeader/index.js (hunk 1 above)
Comment (tonybach) [reply cdap-ui/app/cdap/components/EntityListView/EntityListHeader/index.js]: Yep, will update it now.

**Review by ajainarayanan — APPROVED** [review]
👍

**Pull request merged**